en/docs/guides/identity-lifecycles/enable-account.md

```diff
@@ -1,71 +1,169 @@
+    ![enable-locking](../../../assets/img/guides/enable-locking.png) 
+
+    !!! tip
+    
+        If a user is assigned the **Internal/system** role, the user can
+        bypass account locking even if the user exceeds the specified number
+        of **Maximum Failed Login Attempts**.
+    
+        ??? note "Click here to see more information"
+    
+            WSO2 Identity Server has the **Internal/system** role configured by
+            default. But generally a new user is not assigned the
+            **Internal/system** role by default. Required roles can be assigned
+            to a user depending on the set of permission a user needs to have.
+            For more information on roles and permission, see [Configure Roles
+            and Permissions](../../../guides/identity-lifecycles/manage-roles-overview).
+    
+            Although the **Internal/system** role is configured by default in
+            WSO2 Identity Server, you can delete the role if necessary. To allow
+            users with the **Internal/system** role to bypass account locking,
+            you need to ensure that the role exists in WSO2 Identity Server.
+    
+7.  To enable account locking and disabling for other tenants, log out
+    and repeat the steps given above from [step
+    2](#accountdisabling) onwards.
+
+
+### Disable an account
+
+Follow the steps below to disable a user account through the WSO2 IS
+management console.
+
+1.  Navigate to **Main** > **Identity** > **Claims** > **List** and click on `http://wso2.org/claims`.
+
+2.  Edit the **Account Disabled** claim. See [Edit Claim
+    Mapping](../../../guides/dialects/edit-claim-mapping/) for more information on how to do
+    this.
+
+3.  Tick the checkbox **Supported by Default** and click **Update.**  
+    ![click-supported-by-default](../../../assets/img/guides/click-supported-by-default.png) 
+4.  Navigate to **Main** > **Identity** > **Users and Roles** > **List** > **Users** and click on
+    **User Profile** of the user account that you want to disable.
+5.  Tick the **Account Disabled** checkbox and click **Update**.
+
+!!! tip
+    
+    **Alternatively,** instead of using the management console, you can also
+    enable/disable the user account using the
+    `         setUserClaimValues        ` method in the
+    `                   RemoteUserStoreManagerService                 `
+    after you have configured WSO2 IS for account disabling.
+    
+
+### Send email notifications for account disabling
+
+Once you have configured WSO2 Identity Server for account disabling, you
+can also configure the WSO2 IS to send an email to the user's email
+address when the user account is disabled. To configure this, follow the
+steps below.  
+
+1.  Enable the email sending configurations of the WSO2 Identity Server
+    as explained [here](../../../deploy/configure-email-sending).
```

Comment:
since this is a mandatory step let's have the content inline by using the already available fragment